src/applications/personalization/connected-accounts/components/AppDeletedAlert.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+export class AppDeletedAlert extends React.Component {
+  componentDidMount() {
+    this.alert.focus();
+  }
+
+  alertMessage() {
+    // eslint-disable-next-line prettier/prettier
+    return `${this.props.account.attributes.title} has been disconnected from your account.`;
```

Comment:
@juliaelman I forgot to ask you about the text before I started this. Let me know if you think I should change this (or the title of the alert, "Account Disconnected")